libnmstate/nm/ipv6.py

```diff
@@ -116,6 +120,7 @@ def create_setting(config, base_con_profile):
         setting_ip.props.method = (
             nmclient.NM.SETTING_IP6_CONFIG_METHOD_LINK_LOCAL)
 
+    nm_route.add_routes(setting_ip, config.get('routes', []))
```

Comment:
Should this be a constant? `routes` I mean.

Reply:
Changed to _routes as you suggested for meta data name.